src/components/helper/AxiosHelper.tsx

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios'
+import { TransactionPayload } from '../../lib/types'
+import { baseURL } from '../../config'
+
+const apiEp = baseURL + '/transactions'
+
+//get Authorization header
+const getUserToken = () => {
+  const userToken = sessionStorage.getItem('token')
+  const userObj = userToken ? JSON.parse(userToken) : null
+  return userObj
+}
+
+export const getAllTransactions = async () => {
+  try {
+    const user = getUserToken()
+    if (!user) {
```

Comment:
You should probably throw an error when you fail to retrieve a token from sessionStorage rather than returning an object...